Eastern-Research-Group/expert-query
# GFeature/80 etl gis services

## Related Issues:
* [EQ-80](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=865&view=detail&selectedIssue=EQ-80)

## Main Changes:
* Added an ETL sub-process for pulling in _Assessments_ data.
  * This is calling the ATTAINS GIS service for the time being.
* Restructured the application to pass connection pools by reference instead of using global variables.
* Refactored so that clients are obtained and released at the same level of abstraction.

## Steps To Test:
1. Run the ETL application.
2. Check the logs to verify the `profileTest` table and the `assessments` table are updated in parallel.
3. Verify that the logger prints progress updates as data is fetched for the `assessments` table.
4. Inspect the database after the task is complete to verify that the data is accessible by the read-only user.
